hedera-node/hedera-smart-contract-service-impl/src/main/java/com/hedera/node/app/service/contract/impl/exec/QueryModule.java

```diff
@@ -121,4 +118,21 @@ static HederaEvmContext provideHederaEvmContext(
     @Binds
     @QueryScope
     SystemContractOperations bindSystemContractOperations(QuerySystemContractOperations querySystemContractOperations);
+
+    @Provides
+    @QueryScope
+    static EvmFrameStateFactory provideEvmFrameStateFactory(
+            @NonNull final EvmFrameStates evmFrameStates,
```

Comment:
Since this is always `EvmFrameStates.DEFAULT`, we might stop injecting it here and just use `DEFAULT.from(...)` on line 129 below.

Reply:
Agreed. changed. Thanks